src/styles/blocks/stars.scss

```
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background: url(../images/star.svg) no-repeat center;
+    margin-right: 4px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+}
+
+.stars--0 .stars__star:nth-child(-n),
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
```

use the power of nesting
```suggestion

  &--0 .stars__star:nth-child(-n),
  //...
}
```